### ISSUE TITLE
rule "{indent: [2, 4, {indentSwitchCase: true}]}" is reported invalid

### ISSUE THREAD

**Issue by denghongcai** [eslint/eslint#3786]
cli command:
```
eslint --no-eslintrc --rule "{indent: [2, 4, {indentSwitchCase: true}]}" test.js
```

error output:
```
C:\Program Files\nodejs\node_modules\eslint\lib\config-validator.js:98
        throw new Error(message.join(""));
        ^

Error: CLI:
        Configuration for rule "indent" is invalid:
        Value "data["2"].indentSwitchCase" has additional properties.
```

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by denghongcai** [comment]
eslint version is 1.4.1

**Comment by mysticatea** [comment]
The `indent` rule has a breaking change at 1.0.0.

Please see http://eslint.org/docs/rules/indent#options

Hmm, I could not find it in [Migrating to v1.0.0](http://eslint.org/docs/user-guide/migrating-to-1.0.0)

**Comment by egormerkushev** [comment]
Works for me `"indent": [
      "error",
      2,
      {
        "SwitchCase": 1
      }
    ],`
